packages/message-relayer/src/exec/run.ts

```diff
@@ -69,11 +96,17 @@ const main = async () => {
     l2RpcProvider: l2Provider,
     addressManagerAddress: ADDRESS_MANAGER_ADDRESS,
     l1Wallet: wallet,
```

Comment:
Once we update all of the deploy config then we can remove the backwards compatible requirement and this will clean up this file a ton

Reply:
I've cleaned up this part of the codebase, was able to remove the `ts-ignore` lines